src/leveled_pclerk.erl

```diff
@@ -439,38 +445,22 @@ grooming_score_test() ->
                                     3,
                                     999999,
                                     #sst_options{},
-                                    true,
                                     true),
-    {ok, PidL3_2NC, _, _} = 
-        leveled_sst:sst_newmerge("test/test_area/ledger_files/",
-                                    "2NC_L3.sst",
-                                    KL3_L3,
-                                    KL4_L3,
-                                    false,
-                                    3,
-                                    999999,
-                                    #sst_options{},
-                                    true,
-                                    false),
-    
-    ME1 = #manifest_entry{owner=PidL3_1},
-    ME1B = #manifest_entry{owner=PidL3_1B},
-    ME2 = #manifest_entry{owner=PidL3_2},
-    ME2NC = #manifest_entry{owner=PidL3_2NC},
```

Comment:
The sst_newmerge/10 test function using to include the CountTombs boolean as the final argument - which defaulted to true (this was defaulted in the code, and wasn't configurable by the user).  As part of the refactoring, eqwalizer complained that this could never be `false`.

There were some other options like this, where experimenting with features the code treated things as options that weren't.  To satisfy eqwalizer, I needed to either make it a legitimate option, or remove the code path.  I chose to remove the code path, and so the tests of penciller clerk when `CountTombs = false` became irrelevent.